docs/tutorials/aws.md

```diff
@@ -70,6 +70,7 @@ apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
   name: external-dns
+  namespace: external-dns
```

Comment:
This requires creating a namespace first, let's keep it simple in the example and just use default.

Reply:
I agree, let's leave out the namespace for at this point.